uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/AdbShellUtils.java

```diff
@@ -134,6 +146,7 @@ public static void enableAutomaticTimeZone() {
      * Disable automatic time zone on the device.
      */
     public static void disableAutomaticTimeZone() {
+        Logger.i(TAG,"Disable automatic time zone on the device..");
```

Comment:
you could also add a log above in the `enableAutomaticTimeZone` method

Reply:
added